fractal-puppy/Engine
# XP and Level

In this pr the player can gain Xp and levels.
To test this:
- Recompile everything (Important!!!!)
- Load Level0-Graveyard scen and play.
- Kill some enemies. When first enemy is killed, the HUD inferor bar will increase. When second enemy is killed a message of level up will appear.

P.D.: Can change the experience given by the enemies in the enemy panel.

![LevelUP](https://user-images.githubusercontent.com/7702260/59311461-db698c80-8ca9-11e9-8459-297e5d9bfbe0.gif)
